helper/ldaputil/ldap.go

```diff
@@ -0,0 +1,28 @@
+package ldaputil
+
+import (
+	"crypto/tls"
+
+	"github.com/go-ldap/ldap"
+)
+
+func NewLDAP() LDAP {
+	return &ldapIfc{}
+}
+
+// LDAP provides ldap functionality, but through an interface
+// rather than statically. This allows faking it for tests.
+type LDAP interface {
```

Comment:
Same ideology as above. This simply wraps the static LDAP methods with an interface so it can be easily subbed for testing.